src/components/calcite-filter/calcite-filter.scss

```diff
@@ -1,66 +1,61 @@
 :host {
   @extend %component-host;
   @extend %component-spacing;
-  display: flex;
-  padding: var(--calcite-spacing-half) var(--calcite-spacing-half);
-  width: 100%;
+  @apply flex w-full;
+  padding: theme("spacing.2");
 }
 
 label {
-  align-items: center;
-  display: flex;
-  margin: 0 var(--calcite-spacing-quarter);
-  overflow: hidden;
-  position: relative;
-  width: 100%;
+  @apply flex 
+    items-center 
+    overflow-hidden 
+    relative w-full;
+  margin: 0 theme("spacing.1");
```

Comment:
`mx-1 my-0` maybe here

Reply:
IDK @asangma can you explain the `--calcite-spacing-quarter` you probably had a reason for that? Or is it just old?

Reply:
@asangma  ^

Reply:
FYI, I moved the margin to `@apply` @paulcpederson

Reply:
From a chat with Alan, we can get rid of all the %compoent-spacing vars.